ui/src/routes.ts

```diff
@@ -29,9 +29,9 @@ export const ROUTES: Routes = {
     layout: "sider",
     path: "/connections",
   },
-  credentialLink: {
+  credentialLinkQR: {
     layout: "fullWidthGrey",
-    path: "/credential-link/:linkID",
+    path: "/credentials/scan-link/:linkID",
```

Comment:
We probably should reduce the size of this logo on large screens. What do u think?

<img width="2560" alt="image" src="https://user-images.githubusercontent.com/15896177/234078276-036350c1-9ca4-4c12-bcb0-2e3d17ada94b.png">

Reply:
I was expecting you to fix this in the layout...

<img width="2560" alt="image" src="https://user-images.githubusercontent.com/15896177/234274454-a3581cef-1ed3-495c-b2c0-ae3740dde288.png">